recipes/perl-data-dump/recipe.yaml

```
@@ -0,0 +1,40 @@
+schema_version: 1
+
+context:
+  name: perl-data-dump
+  version: 1.25
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/G/GA/GARU/Data-Dump-${{ version }}.tar.gz
+  sha256: a4aa6e0ddbf39d5ad49bddfe0f89d9da864e3bc00f627125d1bc580472f53fbd
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install
+
+requirements:
+  build:
+    - make
+  host:
+    - perl=5.32
+  run:
+    - perl>=5.32
+
```

```suggestion
test:
 - perl:
     uses:
        - Data::Dump
        - Data::Dump::FilterContext
        - Data::Dump::Filtered
        - Data::Dump::Trace
```
For this recipe you can just delete `run_test.pl` and use a test block here - I had to make a separate `run_test.pl` due to idiosyncracies with `conda-build` that are not relevant to `rattler-build`.